docker/docker-compose.dev.yml

```diff
@@ -12,6 +12,9 @@ services:
     volumes:
       - ..:/srv/app
     command: "flask run --host=0.0.0.0"
+    healthcheck:
+        test: ["CMD", "nc", "-z", "localhost", "5000"]
```

Comment:
Could set Flask to run on `9000`?

Reply:
It's another protocol (HTTP rather than WSGI) so would avoid reusing the same port for different similar things